.github/workflows/opened-pr-test.yml

```
@@ -12,7 +12,84 @@
+          MONGO_INITDB_ROOT_USERNAME: ${{ secrets.MONGO_ROOT_USERNAME }}
+          MONGO_INITDB_ROOT_PASSWORD: ${{ secrets.MONGO_ROOT_PASSWORD }}
+        ports:
+          - 27017:27017
+
     steps:
+      - name: Elasticsearch connection
+        run: |
+          curl http://127.0.0.1:9200
+
+      - name: MySQL connection health check
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:6: Double quote to prevent globbing and word splitting

(shellcheck)